docs/using.rst

```diff
@@ -717,6 +717,10 @@ The following files are available:
      put it into a safe, however - your server still needs to access
      this file in order for SSL/TLS to work.
 
+.. note:: As of Certbot version 0.29.0, private keys for new certificate
```

Comment:
At least when building the docs locally, the formatting here is a little funky. It looks like:
<img width="769" alt="screen shot 2018-11-28 at 3 45 58 pm" src="https://user-images.githubusercontent.com/6504915/49189832-c0928480-f324-11e8-992c-79e1f734ea0e.png">